.github/workflows/lint-test-sdk.yml

```diff
@@ -19,21 +39,22 @@ jobs:
       - name: Install root dependencies
         run: yarn --frozen-lockfile
         working-directory: ./
-      - name: 'Set up GCP SDK for downloading test data'
-        uses: 'google-github-actions/setup-gcloud@v0'
```

Comment:
this action is deprecated; also gcloud is no longer needed for downloading test data

Reply:
🪓